security_groups.tf

```diff
@@ -2,7 +2,10 @@
 ## Gitlab-runner agent security group ##
 ########################################
 
+# ignores: IAM Access Analyzer Not Enabled
```

Comment:
Just wondering, see you have to ignore this on many plces

1. `ignores: ...` Is this just a comment? Or instructing to ignore?
2. Kics-scan ignore-line. Dow e get anything valuable from kics?

Reply:
Will recheck the value of KICS. Seems that it has several false positives.

The `ignores:` is just a comment, so we know in the source what's being ignored.

Reply:
Removed nearly all false positive ignores and configured KICS to ignore these rules. Should be covered anyway by other linters.

Suggest to go ahead with KICS and check how it works in daily life.